### ISSUE TITLE
Missing DECLARE_ACTION_P4 and DECLARE_ACTION_P5

### ISSUE THREAD

**Issue by ghost** [OData/odatacpp-client#11]
In general support in code gen headers for more than 5 params for actions/functions would be good. In meantime though, P4 and P5 declare is missing (but there is corresponding impl) so seems a mismatch/bug

**Comment by PBRCW** [comment]
Yep, IMHO this is a bug. This has been fixed in my fork, maybe you want to try that? You can cone it here:
https://github.com/PBRCW/odatacpp-client
